pkg/logs/config/integration_config.go

```diff
@@ -66,6 +69,8 @@ type IntegrationConfigLogSource struct {
 	Tags            string
 	TagsPayload     []byte
 	ProcessingRules []LogsProcessingRule `mapstructure:"log_processing_rules"`
+
+	Tracker *status.Tracker
```

Comment:
I am wary of having the status tracker inside the source config. This concern is strictly orthogonal and could probably be refactored away here. How would you feel having it one level higher, eg components that embed a source config could embed a tracker as well ?

Reply:
Thanks for your feedback, as our short term plan is to us the auto discovery from the agent, I won't fix this now because IntegrationConfigLogSource is very likely to change. I added this task to our board though.